mc-sema/cfgToLLVM/x86Instrs_Misc.cpp

```diff
@@ -709,6 +709,141 @@ static InstTransResult translate_LEA64_32r(NativeModulePtr natM, BasicBlock *&bl
     return doLeaRef<32>(natM, block, ip, inst);
 }
 
+static InstTransResult translate_CPUID32(NativeModulePtr natM, BasicBlock *&block, InstPtr ip, MCInst &inst) {
```

Comment:
Its a good idea in general; the more complexity we can push out of the translator the better. Since CPUID isn't ELF related, but x86 related it would make more sense to have a separate clang-compiled bitcode blob for it (since we'd need the same code in a Win32 and Win64 driver). I was kind of hoping to just replace the whole translation engine like that eventually (a-la the stuff in remill).